features/bevragen/persoon/opschorting-bijhouding/afgevoerde-pl.feature

```diff
@@ -0,0 +1,44 @@
+#language: nl
+
+Functionaliteit: Persoon: Raadplegen van afgevoerde persoonslijst
+
+  Achtergrond:
+    Gegeven de persoon met burgerservicenummer '000000024' heeft de volgende gegevens
+    | anummer (01.10) | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | geslachtsaanduiding (04.10) |
+    | 0123456789      | William           | de                  | Vries                 | 20040526              | M                           |
+    En de persoon heeft de volgende 'inschrijving' gegevens
+    | datum opschorting bijhouding (67.10) | reden opschorting bijhouding (67.20) |
+    | 20220829                             | F                                    |
+    En de persoon heeft een ouder '1' met de volgende gegevens
+    | burgerservicenummer (01.20) | voornamen (02.10) | voorvoegsel (02.30) | geslachtsnaam (02.40) | geboortedatum (03.10) | geslachtsaanduiding (04.10) |
+    | 000000036                   | Corry             | van                 | Zon                   | 19730428              | V                           |
+    En de persoon heeft een 'nationaliteit' met de volgende gegevens
+    | nationaliteit (05.10) | reden opnemen (63.10) | datum ingang geldigheid (85.10) |
+    | 0001                  | 001                   | 20040603                        |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | functie adres (10.10) | datum aanvang adreshouding (10.30) | gemeente van inschrijving (09.10) |
+    | W                     | 20170423                           | 0518                              |
+    En de 'verblijfplaats' heeft de volgende 'adres' gegevens
+    | gemeentecode (92.10) | straatnaam (11.10) | huisnummer (11.20) | postcode (11.60) |
+    | 0518                 | Spui               | 70                 | 1234AA           |
+
+Rule: bij het raadplegen van een persoon met afgevoerde persoonslijst wordt de reden en datum opschorting geleverd en indien gevraagd de aNummer en burgerservicenummer
+
+  Scenario: geraadpleegde persoon is opgeschort met reden "F" (fout) 
```

Comment:
zoals de scenario nu is geschreven wordt de rule niet beschreven. Krijgt je aNummer en burgerservicenummer alleen als je erom vraagt? Krijgt je alleen opschortingBijhouding gegevens als bijv. alleen naam wordt gevraagd?

Reply:
ik heb de feature uitgebreid met verschillende voorbeelden van vragen om aNummer en burgerservicenummer, al dan niet in combinatie met andere velden, en vragen om andere velden zonder aNummer of burgerservicenummer